app/views/hiring_staff/organisations/managed_organisations/show.html.haml

```diff
@@ -16,7 +16,7 @@
 
         = f.govuk_check_box :managed_organisations,
           'all',
-          label: { text: t('hiring_staff.managed_organisations.options.all') },
+          label: { text: t("hiring_staff.managed_organisations.options.#{current_organisation.group_type == 'local_authority' ? 'all_schools' : 'all'}") },
```

Comment:
i think my comment from last PR still stands about this logic being here

Reply:
the other instances of this kind i see in this PR of thing pass variableName: helperMethod into the yml and this isnt consistent also